tests/NBallotParserTest.php

```diff
@@ -10,19 +10,42 @@ class NBallotParserTest extends TestCase
     private $alice;
     private $bob;
     private $claire;
+    private $dave;
+    private $eve;
 
     public function setUp() : void
     {
         $this->instance = new NBallotParser();
         $this->alice = new Candidate("A");
         $this->bob = new Candidate("B");
         $this->claire = new Candidate("C");
+        $this->dave = new Candidate("D");
+        $this->eve= new Candidate("E");
     }
 
     public function testParseEmptyString() : void
```

Comment:
I changed the behavior of the empty ballot string case. Is the new behavior reasonable?